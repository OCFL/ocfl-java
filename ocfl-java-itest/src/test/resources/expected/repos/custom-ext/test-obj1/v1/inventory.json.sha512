d423236b7c7aaede3291f533bc94660ce64e9f071a9d974c7764214176ca5fb73f27bfb19021b7b84157dbdce90e942b71ecb1cd7f6f69dc110f472e068a557d  inventory.json
