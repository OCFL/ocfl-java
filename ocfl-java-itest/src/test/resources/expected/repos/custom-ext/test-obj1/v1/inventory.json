{
  "id" : "obj1",
  "type" : "https://ocfl.io/1.1/spec/#inventory",
  "digestAlgorithm" : "sha512",
  "head" : "v1",
  "contentDirectory" : "content",
  "fixity" : { },
  "manifest" : {
    "521b9ccefbcd14d179e7a1bb877752870a6d620938b28a66a107eac6e6805b9d0989f45b5730508041aa5e710847d439ea74cd312c9355f1f2dae08d40e41d50" : [ "v1/content/file1.txt" ]
  },
  "versions" : {
    "v1" : {
      "created" : "2019-08-05T15:57:53Z",
      "message" : "commit message",
      "user" : {
        "name" : "Peter",
        "address" : "peter@example.com"
      },
      "state" : {
        "521b9ccefbcd14d179e7a1bb877752870a6d620938b28a66a107eac6e6805b9d0989f45b5730508041aa5e710847d439ea74cd312c9355f1f2dae08d40e41d50" : [ "file1.txt" ]
      }
    }
  }
}